src/shipit_code_coverage/tests/test_artifacts.py

```diff
@@ -59,3 +60,25 @@ def add_dir(files):
 
     with pytest.raises(Exception, message='suite and chunk can\'t both have a value'):
         a.get(chunk='xpcshell-7', suite='mochitest')
+
+
+@mock.patch('shipit_code_coverage.taskcluster.get_task_artifacts')
+@mock.patch('shipit_code_coverage.taskcluster.download_artifact')
+def test_download(mocked_download_artifact, mocked_get_task, LINUX_TEST_TASK_ARTIFACTS):
```

Comment:
Rename `mocked_get_task` to `mocked_get_task_artifacts`, it can be confusing otherwise.